crates/egui/src/input_state/mod.rs

```diff
@@ -1097,13 +1102,13 @@ impl PointerState {
     /// When tapping a touch screen, this will be `None`.
     #[inline(always)]
     pub fn latest_pos(&self) -> Option<Pos2> {
-        self.latest_pos
+        self.latest_pos.map(|p| p / self.zoom_factor)
     }
 
     /// If it is a good idea to show a tooltip, where is pointer?
     #[inline(always)]
     pub fn hover_pos(&self) -> Option<Pos2> {
-        self.latest_pos
+        self.latest_pos.map(|p| p / self.zoom_factor)
```

Comment:
This all feels like the wrong solution to me.

In egui, everything is in the same global coordinate space called _ui points_.

If `self.latest_post` needs dividing with `zoom_factor`, that's a bug in how `latest_pos` is set

Reply:
Thanks for your eyes. Indeed my actual problem was already solved, I just hadn't looked around enough. Closing this MR and the linked issue now.